packages/shared/package.json

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
Is the shared package really something we would want to publish? Has it worth on its own?

Reply:
Yes, we need it in all the other packages. It is a shared central library. This has more value internally than externally but there is a huge benefit for this. We would need to include the shared files in all other packages which would result in code duplication.